launch/realsense_D4xx_nodelet.launch

```diff
@@ -0,0 +1,122 @@
+<launch>
+  <arg name="camera_name"         default="camera" />
+  <arg name="manager"             default="realsense2_camera_manager"/>
+  <arg name="serial_no"           default=""/>
+  <arg name="json_file_path"      default=""/>
+  <arg name="depth"               default="depth"/>
+  <arg name="infra1"              default="infra1"/>
+  <arg name="infra2"              default="infra2"/>
+  <arg name="rgb"                 default="color"/>
+  <arg name="fisheye"             default="fisheye"/>
+  <arg name="accel"               default="accel"/>
+  <arg name="gyro"                default="gyro"/>
+
+  <arg name="fisheye_width"       default="640"/>
+  <arg name="fisheye_height"      default="480"/>
+  <arg name="enable_fisheye"      default="true"/>
+
+  <arg name="depth_width"         default="640"/>
```

Comment:
Can we improve the FOV by using a 16:9 resolution? https://github.com/IntelRealSense/librealsense/wiki/D400-Series-Visual-Presets

Whats here now will limit our FOV.